### DISCUSSION TITLE
SO 2.3.90 installation failed on NUC. Any ideas?

### DISCUSSION THREAD

**Discussion by UserOnion** [Security-Onion-Solutions/securityonion#6370] [Unsupported Versions] [answered]
Trying to install the SO 2.3.90 ( also same error for previous version) on NUC ( see version on the screenshot. Failed with kernel panic. See screenshot. I can install Debian and Ubuntu on the same machine but when I'm trying to install SO with .ISO it failed. 
![Delete_Screen](https://user-images.githubusercontent.com/94726778/143797268-56b6f71d-3ec9-4387-94fc-cc521ab0787c.jpg)

**Comment by dougburks** [comment, marked as answer]
https://docs.securityonion.net/en/2.3/trouble-booting.html

**Reply by UserOnion** [reply]
thank for taking the time and pointing to the document. Before posting anything I'm trying all possible combinations and solutions which I can find on official and google-search docs. After that only I'm asking experts.

**Reply by dougburks** [reply]
If you've followed all the steps at https://docs.securityonion.net/en/2.3/trouble-booting.html and the ISO image still fails, then you'll either need to try different hardware OR install our components on top of standard CentOS 7 64-bit or Ubuntu 18.04 64-bit.
